apps/condo/domains/common/constants/style.js

```diff
@@ -46,6 +46,7 @@ const warningBg = '#FFECEB'
 const warningText = '#FF3B30'
 const infoIconColor = '#39CE66'
 const warningAlert = 'rgba(240, 134, 51, 0.1)'
+const tableRowHoverGray = '#FAFAFA'
```

Comment:
What's this hex is about?

Reply:
this is the color of the ant table row on hover